src/bundles/files/actions.js

```diff
@@ -66,7 +66,7 @@ const cumulativeSize = async (ipfs, cidOrPath) => {
  * @returns {string}
  */
 // TODO: use sth else
-export const realMfsPath = (path) => {
+const realMfsPath = (path) => {
```

Comment:
this is used by files/modals/Modals.js: `import { realMfsPath } from '../../bundles/files/actions.js'`